### PR TITLE
fix: fixed modal box shadow above page content

### PR DESCRIPTION
**Pull request by mehaknasir**
Ticket: https://openedx.atlassian.net/browse/TNL-8405

Issue: The issue was in the modal component in which the box-shadow was appearing under the child component.  The issue is fixed now by removing box-shadow and adding before and after pseudo-element in the modal body class with the linear gradient.
Screenshot 2021-06-22 at 8 48 32 PM

Screenshot 2021-06-22 at 8 47 57 PM

### CODE REVIEW

File: src/Modal/ModalDialogBody.jsx [hunk 1]
```diff
@@ -24,7 +24,7 @@ const ModalDialogBody = ({
     (
       <>
         <div ref={topSentinelRef} />
-        <div className="pgn__modal-body-content">
+        <div className="pgn__modal-body-content" style={{ 'z-index': '-1', position: 'relative' }}>
```
Comment (mehaknasir) [inline src/Modal/ModalDialogBody.jsx:27]: @abutterworth is it okay to pass inline css or should I create a class for the with these properties and pass that here?

**Review by abutterworth — CHANGES_REQUESTED** [review]
Looks good! But, let's add these styles to the SCSS instead of inline `.pgn__modal-body-content {...}` Maybe here: https://github.com/edx/paragon/blob/master/src/Modal/_ModalDialog.scss#L146

File: src/Modal/ModalDialogBody.jsx (hunk 1 above)
Comment (abutterworth) [reply src/Modal/ModalDialogBody.jsx:27]: Oh yes, sorry see my note below

**Comment by abutterworth** [comment]
I just discovered an issue in testing: It looks like by moving the z-index lower we have made the modal body content non-interactive. I'm not sure if this strategy will work for us...

**Review by awaisdar001 — CHANGES_REQUESTED** [review]
I don't know how these changes are going to affect the UI, can you add before/after pictures?  also link the paragon docs?

**Comment by abutterworth** [comment]
@awaisdar001 fyi you can see the deploy preview here: 
image

**Review by abutterworth — APPROVED** [review]
LGTM! Nice work.

**Comment by mehaknasir** [comment]
> I don't know how these changes are going to affect the UI, can you add before/after pictures? also link the paragon docs?

@awaisdar001 

https://deploy-preview-757--paragon-edx.netlify.app/components/stepper/

we have this link in PR deploy from where we can see details I have attached it for you. I am updating description as well.

**Review by awaisdar001 — APPROVED** [review]

**Pull request merged**